srcs/layouts/latn_qwerty_cy.xml

```diff
@@ -0,0 +1,39 @@
+<?xml version="1.0" encoding="utf-8"?>
+<keyboard name="QWERTY (Welsh)" script="latin">
+  <row>
+    <key key0="q" key2="1" key4="loc esc"/>
+    <key key0="w" key1="~" key2="2" key3="\@" key4="ŵ"/>
+    <key key0="e" key1="!" key2="3" key3="\#" key4="ê"/>
+    <key key0="r" key2="4" key3="$"/>
+    <key key0="t" key2="5" key3="%"/>
+    <key key0="y" key2="6" key3="^" key4="ŷ"/>
+    <key key0="u" key2="7" key3="&amp;" key4="û"/>
+    <key key0="i" key2="8" key3="*" key4="î"/>
+    <key key0="o" key2="9" key3="(" key4="ô"/>
+    <key key0="p" key2="0" key3=")"/>
+  </row>
+  <row>
+    <key key0="a" key1="loc tab" key2="`"/>
+    <key key0="s" key2="loc §" key3="loc ß"/>
+    <key key0="d"/>
+    <key key0="f"/>
+    <key key0="g" key2="-" key3="_"/>
+    <key key0="h" key2="=" key3="+"/>
+    <key key0="j" key4="}" key3="{"/>
+    <key key0="k" key3="[" key4="]"/>
+    <key key0="l" key2="|" key3="\\"/>
+    <key key0="accent_circonflexe" key2="accent_trema" key3="accent_grave" />
```

Comment:
You have the dead key and the accented letters at the same time on the layout. This spot could be used for an other dead key or for '.

Reply:
It's the same logic as the Hawaiian layout, the circumflex is the most common diacritic in Welsh and it just gives multiple options to the user to type it.